packages/manager/src/features/Linodes/LinodesCreate/SelectBackupPanel.tsx

```diff
@@ -57,7 +52,7 @@ interface Props {
 
 type CombinedProps = Props & WithProfileProps;
 
-const SelectBackupPanel = (props: CombinedProps) => {
+export const SelectBackupPanel = withProfile((props: CombinedProps) => {
```

Comment:
I know it's out of scope, but while were here, can we switch to the hook version of this?

```suggestion
export const SelectBackupPanel =(props: CombinedProps) => {
  const { data: profile } = useProfile();
```

Reply:
Ah yeah good call - done